openrtb_ext/request.go

```diff
@@ -4,7 +4,15 @@ import "encoding/json"
 
 // ExtRequest defines the contract for bidrequest.ext
 type ExtRequest struct {
-	Prebid ExtRequestPrebid `json:"prebid"`
+	Prebid   ExtRequestPrebid   `json:"prebid"`
+	Currency ExtRequestCurrency `json:"currency"`
```

Comment:
I discussed this with @bretg and we agreed on adding this new field for currency because bidrequest.cur takes an array of currencies instead of a single one. this might be confusing because we are just accepting one currency. another option would be to use bidrequest.cur but throw an error if more than one currency is set? thoughts?

Reply:
I like the idea of using `bidrequest.cur` and throwing an error (return 400) if more than one currency is set.

Worst case, a publisher sends a `len(cur) > 1` and they see a 4xx with a message like "request.cur must have exactly one element. For more info, see {link/to/currency/docs}".

It should be a very simple fix for them, and it makes for a much smoother integration with all the OpenRTB-complaint bidders out there. If we make a special field, every Bidder needs to know about it specially.

Reply:
we can move "adServerCurrency" to bidrequest.cur, but "granularityMultiplier" and "rates" can still be specified in the bid request.